change/@fluentui-react-charting-e365c5ce-592e-422f-9329-1bee461a0dc3.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29324&commitId=608d9a56fa949f16a856a8bb347ba6979c70e518&env=prod&clientType=FLUENTUI)


  <details><summary><b>Nav</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Nav.Root.hover.chromium.png|586| Changed|
|Nav.Root - RTL.default.chromium.png|585| Changed|
|Nav.Root - RTL.click.chromium.png|426| Changed|
|Nav.Root - RTL.hover.chromium.png|584| Changed|
|Nav.Root.default.chromium.png|587| Changed|
|Nav.Root.click.chromium.png|429| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>